### PR TITLE
goalie: cron is failing due to no user in some events

### PR DESCRIPTION
**Pull request by benjdlambert**
PR to check that user is defined. Looks like some events there is no user...

### CODE REVIEW

File: .github/workflows/goalie.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # on a review from someone in the reviewers group, remove the awaiting-review label and add the awaiting-author label
 name: 'goalie: run cron'
-on:
-  schedule:
-    - cron: '* * * * *'
+#on:
+#  schedule:
+#    - cron: '* * * * *'
```
Comment (jhaals) [inline .github/workflows/goalie.yaml:5]: cron schedule no longer needed?

Comment (benjdlambert) [reply .github/workflows/goalie.yaml:5]: It's just so I can run it as a test in the PR

Comment (benjdlambert) [reply .github/workflows/goalie.yaml:5]: Will remove once this passes

Comment (jhaals) [reply .github/workflows/goalie.yaml:5]: 🧙

**Review by freben — APPROVED** [review]
Sure, but prettier is angry!

**Comment by benjdlambert** [comment]
I'm out for a bit now, can you run prettier and merj @freben? I'm getting emails every time the cron fails cos there's a PR with a commit that doesn't have an author just a committer 😅

**Comment by freben** [comment]
on it!

**Review by freben — APPROVED** [review]

**Pull request merged**